src/apollo/queries.js

```diff
@@ -734,17 +734,6 @@ export const TOKEN_CHART = gql`
       dailyVolumeETH
       dailyVolumeToken
       dailyVolumeUSD
-      mostLiquidPairs {
```

Comment:
This attribute is not declared for the type in the schema which was causing, maybe its dynamic or they just didn't update the repo with the new schema

Reply:
Weird, does it exists here https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v2 ?
Anyway seems like a feature we don't need currently

Reply:
Yes it exists, it was preventing the charts from rendering since it was returning an error, i will try to see why